### PR TITLE
Xr llama dev

### PR DESCRIPTION
**Pull request by Xinchunran**
I have add llama manager. Also I add the llama2 for write prd skill and llama analysis

### CODE REVIEW

File: solidgpt/src/workskill/skills/llama_write_prd.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+from solidgpt.src.manager.llamanager import LLAManager
+from solidgpt.src.manager.promptresource import PRODUCT_MANAGER_BRAINSTORM_OUTPUT_TEMPLATE, PRODUCT_MANAGER_BRAINSTORM_ROLE_ASSUMPTION, PRODUCT_MANAGER_PRD_OUTPUT_TEMPLATE, PRODUCT_MANAGER_PRD_ROLE_ASSUMPTION, build_gpt_prompt
+from solidgpt.src.util.util import *
+from solidgpt.src.workskill.workskill import *
+
+class WritePRD(WorkSkill):
```
Comment (wwwwww1020) [inline solidgpt/src/workskill/skills/llama_write_prd.py:6]: Please rename to WritePRDLlama etc to point out it is an llama2 skill

File: solidgpt/src/workskill/skills/llama_analysis.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+from solidgpt.src.manager.promptresource import PRODUCT_MANAGER_5H2W_OUTPUT_TEMPLATE, PRODUCT_MANAGER_ANALYSIS_ROLE_ASSUMPTION, PRODUCT_MANAGER_BRAINSTORM_OUTPUT_TEMPLATE, PRODUCT_MANAGER_BRAINSTORM_ROLE_ASSUMPTION, PRODUCT_MANAGER_PRD_OUTPUT_TEMPLATE, PRODUCT_MANAGER_PRD_ROLE_ASSUMPTION, build_gpt_prompt
+from solidgpt.src.util.util import *
+from solidgpt.src.workskill.workskill import *
+
+Cache_Label_Repos_Summary = "ReposSummary"
+Cache_Label_Repos_Schema = "ReposSchema"
+Cache_Label_Requirements = "Requirements"
+Cache_Label_Product_Analysis = "ProductAnalysis"
+
+class ProductAnalysis(WorkSkill):
```
Comment (wwwwww1020) [inline solidgpt/src/workskill/skills/llama_analysis.py:11]: same below

File: solidgpt/src/configuration/Configuration.yaml [hunk 3]
```diff
@@ -6,4 +6,8 @@ openai_model: gpt-3.5-turbo-16k
 notion_api_key : 
 notion_page_id :
 
-azure_blob_connection_string:
\ No newline at end of file
+azure_blob_connection_string:
+
+#TODO: This method will be moved to api.py to let user input their huggingface config
```
Comment (wwwwww1020) [inline solidgpt/src/configuration/Configuration.yaml:11]: We can remove this comment

File: solidgpt/src/workskill/skills/llama_write_prd.py (hunk 1 above)
Comment (Xinchunran) [reply solidgpt/src/workskill/skills/llama_write_prd.py]: Renamed to WritePRDLlama

File: solidgpt/src/workskill/skills/llama_analysis.py (hunk 2 above)
Comment (Xinchunran) [reply solidgpt/src/workskill/skills/llama_analysis.py]: Renamed to the ProductAnalysisLlama

File: solidgpt/src/configuration/Configuration.yaml (hunk 3 above)
Comment (Xinchunran) [reply solidgpt/src/configuration/Configuration.yaml]: removed the comment

**Review by wwwwww1020 — APPROVED** [review]

**Review by daviddai1998 — APPROVED** [review]

**Pull request merged**